api/envoy/extensions/filters/http/bandwidth_limit/v3/bandwidth_limit.proto

```diff
@@ -72,9 +72,11 @@ message BandwidthLimit {
   //
   // .. note::
   //
-  //   * If set true, the response trailers ``bandwidth-request-delay-ms`` and ``bandwidth-response-delay-ms`` will be added, prefixed by ``response_trailer_prefix``.
-  //   * bandwidth-request-delay-ms: delay time in milliseconds it took for the request stream transfer.
-  //   * bandwidth-response-delay-ms: delay time in milliseconds it took for the response stream transfer.
+  //   * If set true, the following 4 trailers will be added, prefixed by ``response_trailer_prefix``.
+  //   * bandwidth-request-delay-ms: delay time in milliseconds it took for the request stream transfer including request body transfer time and the time added by the filter.
+  //   * bandwidth-response-delay-ms: delay time in milliseconds it took for the response stream transfer including response body transfer time the time added by the filter.
+  //   * bandwidth-request-filter-delay-ms: delay time in milliseconds that added by the filter.
+  //   * bandwidth-response-filter-delay-ms: delay time in milliseconds that added by the filter.
```

Comment:
The bullets after this one should not be bullet points, or be separated from the above 4.

Reply:
updated.